Source/Core/Core/PowerPC/JitInterface.cpp

```diff
@@ -263,4 +265,32 @@ void Shutdown()
     g_jit = nullptr;
   }
 }
+
+void LogCompiledInstructions()
+{
+  // if we run no JIT, there are no compiled instructions to log
+  if (!g_jit)
+    return;
+  static unsigned int time = 0;
+
+  File::IOFile f(StringFromFormat("%sinst_log%i.txt", File::GetUserPath(D_LOGS_IDX).c_str(), time),
+                 "w");
+  File::IOFile g(StringFromFormat("%sinst_not%i.txt", File::GetUserPath(D_LOGS_IDX).c_str(), time),
```

Comment:
This should early return if either IOFile instance fails to open. `g` and `f` are arguably poor naming choices.